databricks/sdk/dbutils.py

```diff
@@ -163,6 +163,35 @@ def listScopes(self) -> typing.List[SecretScope]:
         return [SecretScope(v.name) for v in self._api.list_scopes()]
 
 
+class _JobsUtil:
+    """Remote equivalent of jobs util"""
+
+    class _TaskValuesUtil:
+        """Remote equivalent of task values util"""
+        values = {}
+
+        def get(self, taskKey: str, key: str, default: any = None, debugValue: any = None) -> None:
+            """
+            Returns the latest task value that belongs to the current job run
+            """
+            return debugValue
+
+        def set(self, key: str, value: any) -> None:
+            """
+            Sets a task value on the current task run
+            """
+            self.values[key] = value
+
+        def checkTaskValue(self, key: str, value: any) -> bool:
+            """
+            Check that the task value with given key is equal to the given value
+            """
+            return self.values.get(key) == value
```

Comment:
We should make sure _TaskValuesUtil implements the interface defined at https://github.com/databricks/databricks-sdk-py/blob/main/databricks/sdk/runtime/dbutils_stub.py#L197. This only defines get and set. Otherwise, people may use this method and be surprised that it doesn't exist in DBR.